{"name": "test_login_3[3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <scripts.test_fixture.TestLogin object at 0x000001A83A8380B8>, data = 3\n\n    def test_login_3(self, data):\n        \"\"\"记住密码登录\"\"\"\n        allure.attach(\"输入用户名\", \"输入用户名username_3\")\n        print(\"输入用户名\")\n        allure.attach(\"输入密码\", \"输入用户名password_3\")\n        print(\"输入密码\")\n        allure.attach(\"点击登录\", \"登录成功\")\n        print(\"点击登录\")\n        print(data)\n>       assert False\nE       assert False\n\nscripts\\test_fixture.py:40: AssertionError"}, "description": "记住密码登录", "attachments": [{"name": "输入用户名username_3", "source": "0531a644-bfbc-47e8-bb44-0ef6402b01df-attachment.attach"}, {"name": "输入用户名password_3", "source": "f8abab11-1520-472c-b238-90554c6e07a1-attachment.attach"}, {"name": "登录成功", "source": "ac87f076-577c-45ae-9736-9e34c0346d8c-attachment.attach"}], "parameters": [{"name": "data", "value": "3"}], "start": 1575025869199, "stop": 1575025869200, "uuid": "2487799d-a05d-4b14-924a-6d7766adf3f6", "historyId": "c19157f3bc16370952e1078628fc2183", "testCaseId": "133a8e3ccd283ba22e290157fe855949", "fullName": "scripts.test_fixture.TestLogin#test_login_3", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-47A72RD"}, {"name": "thread", "value": "12696-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture"}]}